ui/src/components/SampleView/shapes.js

```diff
@@ -173,14 +173,14 @@ export function makeText(x, y, fontSize, color, text) {
   });
 }
 
-export function makeScale(height, scaleLengthX, scaleLengthY, color, text) {
+export function makeScale(height, scaleLengthX, scaleLengthY, text) {
   return [
     makeLine(
       10,
       height - 10,
       scaleLengthX + 10,
       height - 10,
-      'green',
+      'rgba(0, 255, 255, 1)',
```

Comment:
Totally ignorant of context here, but it looks like hardcoding rgb values in code, as opposed to using some kind of symbol, This presumably has been considered.

Reply:
`green` is just an alias in this library (CSS/HTML for a green color)  and in this library `fabric.js` for `rgba(0, 255, 0)` so there is no difference really. At least with `rgb` I know more or less which color that is meant :).

Reply:
If you are very interested https://en.wikipedia.org/wiki/Web_colors

Reply:
Cool.